configure/CMakeLists.txt

```diff
@@ -50,11 +50,11 @@ if(OMNI_BASE)
         link_directories(${OMNI_BASE}/lib/x86_win32/)
         include_directories($ENV{OMNI_BASE}/include)
         if (CMAKE_BUILD_TYPE STREQUAL "Debug")
-            set(OMNIORB_PKG_LIBRARIES_DYN "omniORB4_rtd.lib;omniDynamic4_rtd.lib;omnithread_rtd.lib;COS4_rtd.lib;msvcstubd.lib")
```

Comment:
Well, my rationale was that the official omniorb documentation does not list them (see README.win32.txt) and there are no undefined references without them. I reckon they are just there for backwards compatbility

> 
> Why did you remove `msvcstub`? According to omniORB mailing list, it is needed as a workaroud for some MSVC bug.

Well, of the two links you posted, one ends with asking when and why they are needed: "All seems to be ok (including memory usage) at the moment, but I would like to see some official clarification of when msvcstub and msvcstubd are required."
The other only says that you should link to the stub "variants" unless you need the "dynamic interface", which I guess is the full interface: "Because of a bug in VC++, unless you require the dynamic interfaces, it is suggested that you use a smaller stub library called msvcstub.lib. "
It's all super vague, but here's something I found in one of the current examples:
```
# An alternative is to replace the dynamic library with the much smaller 
# library msvcstub.lib. The smaller library contains nothing but stubs
# for the required functions. This is enough when none of the dynamic
# interfaces are used. We use the small library here. If you prefer
# to link with the dynamic library, swap the comment on the next 2
# lines.
```
So really, it seems to be a slimmed-down version for when you want to omit certain functionality in omniORB. If you link it in addition to the others, it is just plain wrong, but it will probably do no harm if linked last (when all undefined references are already resolved by the other libs)

Reply:
Thanks! So `msvcstub` is just a stub to be used in place of `omniDynamic4_rt.lib` if you do not need the full implementation of the latter.

In omniORB source tarball there is a file `src/lib/omniORB/orbcore/msvcdllstub.cc` with below comment. Note that we do not support MSVC++ 5.0 anyway, so is ok to remove the stub.

```
//	The only purpose of this file is to make it possible to link
//      the orbcore DLL under MSVC++ 5.0. None of these functions are
//      used in the DLL. 
//      They are required because MSVC++ generates references to these
//      functions when it generates the destructors for the template 
//      sequence classes that are defined in the headers *even though*
//      the destructors are defined as inline. Call this a feature or
//      a bug I don't care!
```